torch/nn/modules/rnn.py

```diff
@@ -24,11 +24,13 @@ def apply_permutation(tensor, permutation, dim=1):
 
 class RNNBase(Module):
     __constants__ = ['mode', 'input_size', 'hidden_size', 'num_layers', 'bias',
-                     'batch_first', 'dropout', 'bidirectional']
+                     'batch_first', 'dropout', 'bidirectional',
+                     'cat_layer_fwd_bwd_states']
```

Comment:
This is a very descriptive but quite awkward name, would be good to shorten it. If it's really a boolean option (there can never be >2 methods, even in the future), then perhaps `concat=True`. I noticed `tf.keras.layers.Bidirectional` uses `merge_mode='concat'` (and has >2 options).

Reply:
@andfoy any thoughts on this? I suspect other modes could be added in the future, so `merge_mode` would probably make sense here.

Reply:
> so merge_mode would probably make sense here.

I think this is an adequate solution, but I don't know if we should include it as part of this PR, or leave it for a refactoring PR after this one?

Reply:
After the rename to `concat=True` this API looks fine. Maybe `merge_mode` is still an improvement, but not sure. Changing to string/enum here would be a bit of work, so let's see if anyone else has an opinion.